fedireads/incoming.py

```diff
@@ -129,21 +130,22 @@ def handle_follow(activity):
         # Duplicate follow request. Not sure what the correct behaviour is, but
         # just dropping it works for now. We should perhaps generate the
         # Accept, but then do we need to match the activity id?
-        return
+        return True
```

Comment:
I'm not a fan of this pattern of returning True/False to communicate success on the handle methods. Can we use return nothing on success and use exceptions to communicate failure?

Reply:
yeah, I think that's a good idea. I was addressing the linter wanting the methods to have return statements that are all the same type but I think raising vs returning none is a better solution